submissions/kriLEBED/html_css_popup/css/style.css

```diff
@@ -0,0 +1,146 @@
+body {
+    font-family: 'Ubuntu', sans-serif; 
+    color: #858282;
+}
+li {
+    list-style-type: none;
+}
+a {
```

Comment:
would be better to add the class with styles to the link instead of add styles to the tag (it is a good practice). could you please fix it?